proto/sentry_protos/snuba/v1/endpoint_delete_trace.proto

```diff
@@ -0,0 +1,22 @@
+syntax = "proto3";
+
+package sentry_protos.snuba.v1;
+
+import "sentry_protos/snuba/v1/request_common.proto";
+
+// DeleteTraceRequest initiates a delete of all items in a trace. It
+// should also remove downsampled copies.
+//
+// The response will be an acknowledgement of a request, but not
+// a guarantee that the delete has been fulfilled.
+message DeleteTraceRequest {
+  RequestMeta meta = 1;
+  string trace_id = 2;
+}
```

Comment:
I think that it would be better to make a DeleteItemsRequest that takes a TraceItemFilter and a trace_id, so you don't need to write two different endpoints

Reply:
And in the spirit of a batch delete, make a `repeated trace_ids`.